### PR TITLE
Sketch 4

### PR DESCRIPTION
**Pull request by coffeeanddraw**

### CODE REVIEW

File: The_Glowing_Volcano/Assets/Scripts/Door.cs [hunk 1]
```diff
@@ -20,16 +61,39 @@ protected override void Awake()
...
+
+    private void InitializeLocked()
+    {
+        if (key != null)
+            isLocked = true;
     }
 
     public override void InteractWith()
     {
-        if (!isOpen)
```
Comment (ifaith1) [inline The_Glowing_Volcano/Assets/Scripts/Door.cs:27]: There shouldn't be a space by if it should be if(!isOpen)

File: The_Glowing_Volcano/Assets/Scripts/TransportInventoryObject.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    private new Transform transform;
+
+    protected override void Start()
+    {
+        base.Start();
+        transform = objectToTransport.GetComponent<Transform>();
+    }
+
+    //void Update() // check every frame
```
Comment (ifaith1) [inline The_Glowing_Volcano/Assets/Scripts/TransportInventoryObject.cs:27]: Delete unneeded comments

**Review by ifaith1 — CHANGES_REQUESTED** [review]
Everything looks great! There was just a spacing issue and a comment that needed to be deleted.

File: The_Glowing_Volcano/Assets/Scripts/TransportInventoryObject.cs (hunk 2 above)
Comment (coffeeanddraw) [reply The_Glowing_Volcano/Assets/Scripts/TransportInventoryObject.cs:27]: will be deleting the comments

**Review by ifaith1 — APPROVED** [review]

**Pull request merged**